pex/resolve/target_options.py

```diff
@@ -28,11 +32,9 @@ def register(parser):
         type=str,
         action="append",
         help=(
-            "The Python interpreter to use to build the PEX (default: current interpreter). This "
-            "cannot be used with `--interpreter-constraint`, which will instead cause PEX to "
```

Comment:
I'm not following why you removed `--interpreter-constraint` comment?

Reply:
Because it was untrue.